Makefile

```diff
@@ -0,0 +1,7 @@
+build_r:
+	cp LICENSE clients/r-client
+	R CMD build clients/r-client
+
+build_py:
+	cp LICENSE clients/py-client
+	cd py-pkg && python3 setup.py sdist
```

Comment:
can you please add a new line at the end of this file?

Reply:
Sure thing- sorry about that.